examples/create_postcards_from_csv/index.js

```diff
@@ -42,6 +42,11 @@ parse(input, function (err, rows) {
         name: row[0],
         car: row[3],
         mileage: row[4]
+      },
+      metadata: {
+        name: row[0],
+        customer_id: "customer_123",
+        campaign: "Example Campaign"
```

Comment:
nit: change the double quoted strings in these two lines to use single quotes. to be consistent with syntax in the rest of the file